qa/tasks/cephfs/test_misc.py

```diff
@@ -504,6 +504,27 @@ def test_dump_inmemory_log_on_missed_internal_heartbeats(self):
                 return
         self.assertTrue(False, "Failed to dump in-memory logs during missed internal heartbeat")
 
+    def _session_client_evict(self, cmd):
+        mount_a_client_id = self.mount_a.get_global_id()
+        info = self.fs.rank_asok(cmd + ['ls'])
+        self.assertIsNotNone(info)
+        size = len(info)
+        with self.assertRaises(CommandFailedError) as ce:
+            info = self.fs.rank_asok(cmd + ['evict'])
+        self.assertEqual(ce.exception.exitstatus, -errno.EPERM)
+        info = self.fs.rank_asok(cmd + ['evict', f'id={mount_a_client_id}'])
+        self.assertEqual(len(info), size - 1)
+        self.assertNotIn(mount_a_client_id, [val['id'] for val in info])
+        info = self.fs.rank_asok(cmd + ['evict', '--yes-i-really-mean-it'])
```

Comment:
Does the `--yes-i-really-mean-it` command actually have something to evict here after the successful eviction above?

Reply:
yes, we would be having two clients initially. By the time we evict `mount_a_client_id`, we would be left with client on `mount_b`.

Reply:
Yes - that because this test class has `CLIENTS_REQUIRED = 2`.

Reply:
Shifted the test in a separate class.